test/linters/argument_mappers/base_test.rb

```diff
@@ -30,12 +30,12 @@ def test_returns_known_system_arguments_as_string
   end
 
   def test_raises_if_a_class_cannot_be_converted
-    @file = '<div class="mr-1 p-3 d-none d-md-block anim-fade-in text-center"></div>'
+    @file = '<div class="mr-1 p-3 d-none d-md-block anim-fade-in text-fuzzy-waffle"></div>'
```

Comment:
This is an actual class but for some reason was used as an example of a class that could not be converted. I'm not sure why `text-center` was not convertible, but with these changes it does become convertible. If that was a deliberate choice, I can revert.